src/goesproc/config.cc

```diff
@@ -111,6 +112,99 @@ bool loadHandlers(const toml::Value& v, Config& out) {
       }
     }
 
+    auto gradient = th->find("gradient");
+    if (gradient) {
+      auto trs = gradient->as<toml::Table>();
+      for (const auto& it : trs) {
+        auto interpolation = it.second.find("interpolation");
+
+        if (interpolation) {
+          if (0 == interpolation->as<std::string>().compare("hsv")) h.lerptype = LERP_HSV;
+          else if (0 == interpolation->as<std::string>().compare("rgb")) h.lerptype = LERP_RGB;
+          else {
+            out.ok = false;
+            out.error = "Unknown gradient interpolation type (supported types are \"rgb\" and \"hsv\")";
+            return false;
+          }
+        }
+
+        Gradient grad;
+        auto points = it.second.get<toml::Array>("points");
+        for (const auto& point : points) {
+          auto units = point.find("units");
+          if (!units) units = point.find("u");
+
+          // HTML-style #xxxxxx or #xxx color definition
+          auto color = point.find("color");
+          if (!color) color = point.find("c");
+
+          // ... or RGB components (0-1 or 0-255)
+          auto red = point.find("red");
+          if (!red) red = point.find("r");
+          auto green = point.find("green");
+          if (!green) green = point.find("g");
+          auto blue = point.find("blue");
+          if (!blue) blue = point.find("b");
```

Comment:
These are not used.

Reply:
These were put in to allow users to use a variety of color specification types in the config.  My example gradients only use HTML-style colors, but you can also use colors like:
* { r=0.5, g=0.5, b=0.5 }
* { red=255, green=128, blue=64 }
* { hue=0.33,saturation=1,value=0.5}
* { h=355, s=100, v=50}  // hue in degrees, sat and val in percent

If you think support for so many different ways to configure the same thing is overkill, I can remove the aliases.

Should we preserve any option for HSV color points in a gradient definition?

Reply:
I know but the variables themselves are not used, so even if you use them it won't work. Below only the r, g, and b floats are used to compile a gradient point.

I think the written out aliases for both can be removed. In the context of gradients, rgb and hsv should be clear enough for folks changing the config.